SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorConfiguration.java

```diff
@@ -226,6 +226,9 @@
   @JsonProperty
   private boolean useFileAttributes = false;
 
+  @JsonProperty
+  private int healthcheckMaxRetries = 30;
```

Comment:
Maybe name this `default...` instead? Makes it clearer as to why there is a setting here